core/templates/dev/head/pages/exploration_editor/EditorServices.js

```diff
@@ -1713,14 +1665,13 @@ oppia.factory('computeGraphService', [
       var nodes = {};
       var links = [];
       var finalStateIds = [];
-      for (var stateName in states) {
-        var interaction = states[stateName].interaction;
+      for (var stateName in states.getStates()) {
```

Comment:
That _could_ work, but it seems less nice, because "graph data" is a lot fuzzier.